pkgs/development/tools/misc/sccache/default.nix

```diff
@@ -1,17 +1,17 @@
-{ stdenv, lib, fetchFromGitHub, rustPlatform, pkg-config, openssl, Security }:
```

Comment:
Why move `stdenv`?

Reply:
The [documented convention](https://nixos.org/manual/nixpkgs/unstable/#sec-syntax) is to sort in the order used with the exception of lib